using_images/other_images/jenkins.adoc

```diff
@@ -11,9 +11,9 @@
 toc::[]
 
 == Overview
-OpenShift provides a Docker image for running Jenkins.  This image provides
-a Jenkins server instance which can be used to setup continuous testing/integration/delivery
-flows.
+OpenShift provides a Docker image for running Jenkins.  This image provides a Jenkins server instance which can be used to set up a basic flow for continuous testing, integration, and delivery.
+
+Specifically, this sample Jenkins job triggers a new build of a `*BuildConfig*` defined in OpenShift, tests the output of that build, and then on successful build, retags the output to indicate the build is ready for production.
```

Comment:
"This image also includes a sample job which triggers a new build of............"